src/org/labkey/remoteapi/announcements/AbstractMessageThreadCommand.java

```diff
@@ -0,0 +1,20 @@
+package org.labkey.remoteapi.announcements;
+
+import org.json.simple.JSONObject;
+import org.labkey.remoteapi.PostCommand;
+
+public class AbstractMessageThreadCommand extends PostCommand<MessageThreadResponse>
```

Comment:
should this be abstract, or is it useful on it's own?

Reply:
Good catch. I missed this when reviewing.